### ISSUE TITLE
Reverenced precondition missing on new 2.5.3 profiles

### ISSUE THREAD

**Issue by sgmess** [Kav-Simulations/MobiFlight-KavSimulations#16]
Looks like a referenced precondition is missing from the new 2.5.3 configs but included in the z_development ones as below.

```
<config guid="a9c996db-76cb-4e02-98bd-44df3da839a9">
      <active>true</active>
      <description>AC BUS IS ON</description>
      <settings msdata:InstanceType="MobiFlight.OutputConfigItem, MFConnector, Version=10.1.4.0, Culture=neutral, PublicKeyToken=null" xmlns:msdata="urn:schemas-microsoft-com:xml-msdata">
        <source type="SimConnect" VarType="CODE" Value="(L:A32NX_ELEC_AC_ESS_BUS_IS_POWERED)" UUID="95b6f5a3-c4e8-419d-976e-82a018bdec83" />
        <test type="Float" value="1" />
        <modifiers />
        <display type="-" serial="-" trigger="normal" pin="" pinBrightness="255" />
        <preconditions />
        <configrefs />
      </settings>
    </config>
```

**Comment by elral** [comment]
Yes, indeed the config reference for `AC BUS IS ON` at least for the FBWA320 profile is missing.

There are two ways until it get's fixed:
* create a config reference according the attached picture
![image](https://github.com/Kav-Simulations/MobiFlight-KavSimulations/assets/3263285/d36978cd-ddd8-4960-b678-54be18621f20)
* or merge the attached `AC BUS IS ON.mcc` file into your config
[AC BUS IS ON.zip](https://github.com/Kav-Simulations/MobiFlight-KavSimulations/files/14229263/AC.BUS.IS.ON.zip)

**Comment by Jak-Kav** [comment]
Fixed in #17
